keel-web/src/main/kotlin/com/netflix/spinnaker/config/SchedulingConfiguration.kt

```diff
@@ -0,0 +1,10 @@
+package com.netflix.spinnaker.config
+
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty
+import org.springframework.context.annotation.Configuration
+import org.springframework.scheduling.annotation.EnableScheduling
+
+@Configuration
+@ConditionalOnProperty("scheduling.enabled", matchIfMissing = true)
+@EnableScheduling
+class SchedulingConfiguration
```

Comment:
This is a new configuration class that conditionally enables scheduling (previously we just had `@EnableScheduling` on `KeelApplication` so it was always on). If the `scheduling.enabled` property is missing, or equal to `true` scheduling is enabled. So tests can set it to `false` to disable it.

Reply:
I remember trying to debug this myself several months ago and seeing that the annotations didn't take effect, but I also have this vague memory that even the config property wasn't enough, because kork was enabling scheduling programmatically and it was hard-coded. Are you positive this does the trick? If not, I can go dig that code in kork again.

Reply:
Looks like that was removed in this PR: https://github.com/spinnaker/kork/pull/743 🙏

Reply:
Original line was here: https://github.com/spinnaker/kork/blob/44e1536e59fe19aa461b03e73438b18597408ec8/kork-web/src/main/groovy/com/netflix/spinnaker/config/TomcatConfiguration.groovy#L40